projects/ui-components/lf-metadata/lf-field-template-container/lf-field-template-container-demo.service.ts

```diff
@@ -135,54 +147,63 @@ export const fieldInfosPerTemplate: { [templateId: number]: TemplateFieldInfo[]
       name: 'Grandparent Date',
       fieldType: FieldType.Date,
       rule: { ancestors: [] },
+      displayName: 'dynamic grandparent'
     },
     {
       id: DynamicFieldIds.ParentDate,
       name: 'Parent Date',
       fieldType: FieldType.Date,
       rule: { ancestors: [DynamicFieldIds.GrandparentDate] },
+      displayName: 'dynamic parent date'
     },
     {
       id: DynamicFieldIds.ChildDate,
       name: 'Child Date',
       fieldType: FieldType.Date,
       rule: { ancestors: [DynamicFieldIds.ParentDate] },
+      displayName: 'dynamic child date'
     },
     {
       id: DynamicFieldIds.GrandparentTime,
       name: 'Grandparent Time',
       fieldType: FieldType.Time,
       rule: { ancestors: [] },
+      displayName: 'dynamic grandparenttime'
     },
     {
       id: DynamicFieldIds.ParentTime,
       name: 'Parent Time',
       fieldType: FieldType.Time,
       rule: { ancestors: [DynamicFieldIds.GrandparentTime] },
+      displayName: 'dynamic parent time'
     },
     {
       id: DynamicFieldIds.ChildTime,
       name: 'Child Time',
       fieldType: FieldType.Time,
       rule: { ancestors: [DynamicFieldIds.ParentTime] },
+      displayName: 'dynamic child time'
     },
     {
       id: DynamicFieldIds.GrandparentDateTime,
       name: 'Grandparent DateTime',
       fieldType: FieldType.DateTime,
       rule: { ancestors: [] },
+      displayName: 'grandparent datetime'

```

Comment:
why are these displayNames not started with "dynamic"?

Reply:
I missed those, updated